
int8 cam_id 
bool on

---

bool success   
string message 

